Source/WebCore/svg/graphics/filters/SVGFilter.h

```diff
@@ -49,6 +50,10 @@ class SVGFilter final : public Filter {
     
     FilterEffectVector effectsOfType(FilterFunction::Type) const final;
 
+    FilterResults* results() { return m_results.get(); }
+    WEBCORE_EXPORT FilterResults& ensureResults(const FilterResultsCreator& createFilterResults = [] { return makeUnique<FilterResults>(); });
```

Comment:
Either this doesn’t compile, or it does somehow and returns a reference to a deleted object.